### DISCUSSION TITLE
Batch Audio to Text Convert

### DISCUSSION THREAD

**Discussion by voska7** [SubtitleEdit/subtitleedit#5954] [Ideas]
can you please implement batch audio to text convert feature?

**Comment by niksedk** [comment]
I can try - how is this: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.5/SubtitleEditBeta.zip ?
![image](https://user-images.githubusercontent.com/1261331/168647892-8d763489-b8c4-40cb-aac6-fc4958ac9401.png)

![image](https://user-images.githubusercontent.com/1261331/168647963-bd023a09-89fc-4876-bdf6-bc44a44e6b12.png)

**Comment by voska7** [comment]
perfectly worked. i tried 20 files, all worked fine.
